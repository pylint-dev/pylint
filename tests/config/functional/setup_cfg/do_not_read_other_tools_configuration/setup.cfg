[myothertool]
ignore = allthefiles

[pylint.MESSAGES CONTROL]
extension-pkg-whitelist=pygame
enable=
    useless-supression,

disable=
    missing-docstring,
