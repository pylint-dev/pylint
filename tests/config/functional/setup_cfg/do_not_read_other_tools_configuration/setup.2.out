************* Module {abspath}
{relpath}:1:0: E0012: Bad option value for '--enable', expected a valid pylint message and got 'useless-supression' (bad-option-value)
